TESTS/network/wifi/mbed_app.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Would you mind renaming this file to `template_mbed_app.txt`? This will prevent the file from being picked up at all by the build system, but I think that having this reference is still extremely useful.